### ISSUE TITLE
Could you please provide the training code?

### ISSUE THREAD

**Issue by xfcxing** [littlespray/VE-Bench#7]
Hi, thank you for sharing this great project! I’ve gone through the repository and found the inference code very helpful. However, I noticed that the training-related code (e.g., model training scripts, data preprocessing for training, loss functions, etc.) is not included in the repo.

Would it be possible for you to share the training code or give some guidance on how to train the model from scratch?

Any help would be greatly appreciated. Thanks again for your excellent work!

**Comment by littlespray** [comment]
Hey there,

Thank you for your interest in our work. I am sorry that the original training code has become quite messy after many iterations. I will work on cleaning it up as soon as possible, but it might take some time to make it presentable. 

In the meantime, I can send you the raw code first. I hope it can offer a quick glimpse into our key modules.

Besides, I highly recommend refering to DOVER's training code. Our work builds directly on their codebase, and we didn’t modify the core training pipeline—so it might be clearer for your development.
